Sources/VCIClient/VCIClient.swift

```diff
@@ -0,0 +1,107 @@
+import Foundation
+import JWTKit
+
+public class VCIClient {
+    
+    let networkSession: NetworkSession
+    let traceabilityId: String
+    
+    public init(traceabilityId: String, networkSession: NetworkSession? = nil) {
+        self.networkSession = networkSession ?? URLSession.shared
+        self.traceabilityId = traceabilityId
+    }
+    
+    
+    public func getCredential(
```

Comment:
keep function name consistent across all platform. Please check with kotlin, it's `requestCredential`